contracts/src/RealitioForeignArbitrationProxyWithAppeals.sol

```diff
@@ -390,43 +396,23 @@ contract RealitioForeignArbitrationProxyWithAppeals is IForeignArbitrationProxy,
     }
 
     /**
-     * @notice Allows to withdraw any reimbursable fees or rewards after the dispute gets solved for multiple ruling options (answers) at once.
-     * @dev This function is O(n) where n is the number of queried answers.
-     * @dev This could exceed the gas limit, therefore this function should be used only as a utility and not be relied upon by other contracts.
+     * @notice Allows to withdraw any rewards or reimbursable fees for all rounds at once.
+     * @dev This function is O(n) where n is the total number of rounds. Arbitration cost of subsequent rounds is `A(n) = 2A(n-1) + 1`.
+     *      So because of this exponential growth of costs, you can assume n is less than 10 at all times.
```

Comment:
There are extra spaces at the beginning

Reply:
Yeah, I aligned it to above on purpose. Like a paragraph.